odoo/addons/base/views/ir_actions_views.xml

```diff
@@ -325,63 +325,60 @@
                                 help="Run this action manually."/>
                     </header>
                     <sheet>
-                        <div class="oe_title">
-                            <label for="name"/>
-                            <h1><field name="name" placeholder="e.g. Update order quantity"/></h1>
-                        </div>
+                        <h1 class="oe_title">
+                            <field name="name" placeholder="e.g. Mass archive contacts"/>
+                        </h1>
                         <group name="action_wrapper">
                             <group name="action_content">
-                                <field name="model_id" options="{'no_create': True}"/>
                                 <field name="model_name" invisible="1"/>
-                            </group>
-                            <group>
-                                <field name="state"/>
+                                <field name="model_id" invisible="1"/>
+                                <field name="model_id" options="{'no_create': True}" groups="base.group_no_one"/>
```

Comment:
The model is present twice
![Screenshot 2023-09-05 at 16-33-48 Odoo - base automation NewId_0x7fa678bc04c0](https://github.com/odoo/odoo/assets/564822/7afa1278-fb05-4617-80a1-6fc504445ce0)

Reply:
Indeed,
We want to have it in debug mode (with the group)
We also need it all the time because it is used in e.g. invisible evaluation, etc.

But since v16, fields with groups are simply no longer present in the view if you don't have the group => it needs to be included twice (and for 'simple fields', having them present twice is fine)

See e.g. https://github.com/odoo/odoo/pull/95729/files#diff-48cb4309a6006f91b2b40e4c1049860218d419fce1782c7dcc278329803129caR323 or the 'partner_invoice_id' which is present twice in the SO form view for the same reason.

Reply:
In short: not an issue per se. A bit fugly, but the groups changes in v16 make this unavoidable

Reply:
>  We want to have it in debug mode (with the group)

But this view is accessible only in debug mode, no?

Reply:
> But this view is accessible only in debug mode, no?

No, you can go to Automation via Studio (for a long time) and now via the column cog menu entry 'Automations' in a grouped kanban.
![image](https://github.com/odoo/odoo/assets/9530767/0f51006e-0beb-42c9-a2a7-ddbab0a3797b)

> I meant twice as in the automated action and the server action
Same as for server actions which executes several actions: you could have a base automation that runs on crm.lead which an action 'execute python code' that runs on project.task

It's... weird i guess? But not forbidden... 
![image](https://github.com/odoo/odoo/assets/9530767/b9689fd9-b939-4d50-ac1e-77795b4f7e86)